docs/index.md

```diff
@@ -101,6 +101,31 @@ This mod uses the credentials configured in the [Steampipe Azure plugin](https:/
 
 No extra configuration is required.
 
+### Common and Tag Dimensions
+
+The benchmark queries use common properties (like `connection_name`, `resource_group`, `region`, `subscription`, `subscription_id` and `tenant_id`) and tags that are defined in the form of a default list of strings in the `mod.sp` file. These properties can be overwritten in several ways:
+
+- Copy and rename the `steampipe.spvars.example` file to `steampipe.spvars`, and then modify the variable values inside that file
+- Pass in a value on the command line:
+
+  ```shell
+  steampipe check benchmark.cis_v140 --var 'common_dimensions=["resource_group", "region", "subscription", "subscription_id", "tenant_id"]'
```

Comment:
@khushboo9024 should we keep tenant_id? we can remove "subscription_id"? We may align the same to steampipe.spvars.example